lib/rucio/rse/protocols/posix.py

```diff
@@ -210,3 +211,46 @@ def stat(self, pfn):
         """
         path = self.pfn2path(pfn)
         return {'filesize': os.stat(path)[os.path.stat.ST_SIZE], 'adler32': adler32(path)}
+
+
+class Symlink(Default):
+    """ Implementing access to RSEs using the local filesystem, creating a symlink on a get """
+
+    def get(self, pfn, dest, transfer_timeout=None):
+        """ Provides access to files stored inside connected the RSE.
+            A download/get will create a symlink on the local file system pointing to the
+            underlying file. Other operations act directly on the remote file.
+            :param pfn: Physical file name of requested file
+            :param dest: Name and path of the files when stored at the client
+            :param transfer_timeout Transfer timeout (in seconds) - dummy
+            :raises DestinationNotAccessible: if the destination storage was not accessible.
+            :raises ServiceUnavailable: if some generic error occured in the library.
+            :raises SourceNotFound: if the source file was not found on the referred storage.
+         """
+        try:
+            path = self.pfn2path(pfn)
+            os.symlink(path, dest)
+            self.logger(logging.DEBUG,
+                        'Symlink {} created for {} from {}'
+                        .format(dest, path, pfn))
+        except Exception as e:
+            print("JW,", e)
```

Comment:
remove?

Reply:
Remove also the generic exception, which should propagate up higher.